### PR TITLE
Add EFTPOS icon as a payment method

### PR DESCRIPTION
**Pull request by amrdruid**
Addresses Ticket: https://sumupteam.atlassian.net/browse/EPE-528

## Purpose

Add EFTPOS icon as a payment method

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: packages/icons/manifest.json [hunk 1]
```diff
@@ -296,6 +296,11 @@
       "category": "Payment method",
       "size": "24"
     },
+    {
+      "name": "eftpos",
+      "category": "Payment method",
+      "size": "24"
+    },
```
Comment (connor-baer) [inline packages/icons/manifest.json:303]: Please put the icon in alphabetical order, as specified in the [contribution guidelines](https://circuit.sumup.com/?path=/docs/contributing-contributing-icons--docs).

**Review by connor-baer — CHANGES_REQUESTED** [review]
Hi @amrdruid, thanks for suggesting this new icon. Before it can be added to the icon package, it needs to be added to the [Figma library](https://www.figma.com/file/LnhSTiHtYMVw3Ppw5QZVph/Circuit-UI-Foundation?type=design&node-id=2480-3476&t=uP7MCCn2henJkijs-0). Please make a request with the design team to have it added.

**Review by connor-baer — COMMENTED** [review]
Thanks for addressing my feedback @amrdruid. I see that the icon was added in the "[Card schemes](https://www.figma.com/file/LnhSTiHtYMVw3Ppw5QZVph/Circuit-UI-Foundation?type=design&node-id=11896%3A67&t=HDDQB5M83waAPYNb-1)" category. Its design was also adapted to match the other card scheme icons. Can you update the PR to reflect this, please?

**Comment by amrdruid** [comment]
@connor-baer Thanks for catching that! I've added it now, let me know what you think

**Review by connor-baer — APPROVED** [review]

**Pull request merged**